crates/ruff_python_parser/resources/invalid/expressions/unclosed_fstring_lbrace.py

```diff
@@ -0,0 +1,5 @@
+f'{'
+# f'{foo!r'
+# f'{foo='
+# f"{"
+# f"""{"""
```

Comment:
Can you tell me why are these commented out? I guess it's because the parser is panicking otherwise? If so, then even the first one should be commented out for now, right? We can add a `TODO` comment on top to fix these.

Reply:
Oh, I forgot to uncomment 😅

Reply:
Do you expect them to pass with the current state of the parser? I think it'll panic considering that there will be two `Unknown` tokens emitted corresponding to the two errors and it will seem that the parser isn't progressing.

I'm fine with merging with a TODO comment to fix them.

Reply:
> Do you expect them to pass with the current state of the parser?

No, it will still get stuck because of the EOF error returned by the lexer, but if we remove the line that returns that error from the lexer, then it works.